packages/components/src/Chips/tests/InternalChip.test.tsx

```diff
@@ -22,33 +22,9 @@ it("should fire the callback when it's clicked", async () => {
 });
 
 describe("Chip icon colors depending on state", () => {
-  it("should be red when it's invalid", () => {
```

Comment:
Along with the skipped tests at the bottom of this file (that I removed), there is something up with trying to check for the fill color on these icons. Easily the longest time I spent on this PR was trying to get these few tests to pass. I checked the render in the browser, and they render properly there, but for some reason don't render properly in the tests, at least not being able to pull the colours out via toHaveStyle (everything is the base style, it never picks up the cascaded styles). Since everything renders well elsewhere, and because of the skipped tests at the bottom, I'm okay with pulling these tests out.

I took the exact code out, rendered it in the browser, and everything is disabled properly or red properly, but when you render in these tests, everything always comes back as though it's the original render without disabled/invalid.